docs/application/native/guides/ui/dali/animation.md

```diff
@@ -14,7 +14,7 @@ You can use animation to allow your objects to move around and change their prop
 
 DALi provides a rich and easy to use animation framework which allows you to create visually rich applications. The `Dali::Animation` class can be used to animate the [animatable properties](properties.md#attributes) of any number of objects.
 
-DALi animations [occur in a dedicated thread](multi-threaded.md#animations). This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
+DALi animations [occur in a dedicated thread](dali-rendering.md#animations). This allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.
```

Comment:
Please make a minor change in the sentence:

DALi animations [occur in a dedicated thread](dali-rendering.md#animations). It allows animations to run smoothly, regardless of the time taken to process the input, events, and other factors in the application code.

Reply:
Update the file as your comment.